packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Configure protocol specific settings
+---
+
+Your app has to support at least one protocol: SAML or OIDC for authentication, or SCIM for provisioning.
```

Comment:
"Your app has to support at least one protocol" > Maybe, to be more specific: "Your app has to support at least one protocol for interacting with Okta"